tests/e2e/framework/kubernetes.go

```diff
@@ -30,13 +30,15 @@ import (
 )
 
 const (
-	yamlSuffix         = ".yaml"
-	istioInstallDir    = "install/kubernetes"
-	istioAddonsDir     = "install/kubernetes/addons"
-	nonAuthInstallFile = "istio-one-namespace.yaml"
-	authInstallFile    = "istio-one-namespace-auth.yaml"
-	istioSystem        = "istio-system"
-	mixerConfigDefault = "istio-config-default"
+	yamlSuffix           = ".yaml"
+	istioInstallDir      = "install/kubernetes"
+	istioAddonsDir       = "install/kubernetes/addons"
+	nonAuthInstallFile   = "istio.yaml"
+	authInstallFile      = "istio-auth.yaml"
+	nonAuthInstallFileNs = "istio-one-namespace.yaml"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Can you use `NS` or the full spelling `Namespace` instead of `Ns` for namespace, e.g. nonAuthInstallFileNS or nonAuthInstallFileNamespace?

Reply:
Namespace